docs/docs/getting-started/installation/aws.md

```diff
@@ -0,0 +1,127 @@
+---
+title: AWS
+sidebar_label: AWS
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The goal of this document is to provide an overview of how to run Airy Core on
```

Comment:
Right before this, so that @sgarroum is happy :), I would add a TLDR component with content along the lines of "Install Airy Core on AWS with one command"